users/vieting/experiments/switchboard/ctc/feat/network_helpers/perturbation.py

```diff
@@ -122,12 +131,25 @@ def apply_codecs(audio, sample_rate, random_state, codecs):
                     tfm.set_output_format(encoding="u-law")
                 else:
                     raise NotImplementedError(f"Codec {codec} not implemented.")
+        return tfm.build_array(input_array=audio, sample_rate_in=sample_rate)
```

Comment:
Was that not implemented before?

Reply:
No, it was not.

Reply:
As I mentioned earlier, it is generally desirable to change only things that are related to the PR (so here as per your title the non linearity and fixed source code export). In this case, I'm okay with keeping it in here, but please keep this in mind for the future.